src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side1;
+    private int side2;
+
+    public RightTriangle(String color, int side1, int side2) {
+        super(color);
+        this.side1 = side1;
+        this.side2 = side2;
+    }
+
+    public double getArea() {
+        return 0.5 * side1 * side2;
+    }
+
+    @Override
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("\nFigure: RightTriangle, "
```

Comment:
Avoid using System.out.println in the draw method. It's better to return the drawing information as a String and let the caller decide how to handle it.